scheduler.py

```diff
@@ -0,0 +1,8 @@
+import subprocess
+import time
```

Comment:
all the code looks good
add a generic class that launches the entire application, so that it's not scheduler.py